libs/sql-schema-describer/src/postgres.rs

```diff
@@ -1,5 +1,6 @@
 //! Postgres description.
 use super::*;
+use failure::_core::convert::TryInto;
```

Comment:
This is a reexport from the std library, it should be `std::convert::TryInto` (we're going to move away from `failure` soon).